src/services/auth-service.ts

```diff
@@ -0,0 +1,48 @@
+import {CreateUserRequest, LoginRequest} from "../global";
+import {PrismaClient} from '@prisma/client'
+
+const prisma = new PrismaClient()
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const saltRounds = 10;
```

Comment:
prefer these types of constants as ALL CAPS to make it clear where it's coming from